bundle/downstream/bundle.Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM scratch
+
+LABEL com.redhat.delivery.operator.bundle=true
+LABEL com.redhat.delivery.openshift.ocp.versions="v4.7"
+LABEL com.redhat.openshift.versions="v4.7"
```

Comment:
Just so it's understood that this is defining the minimum Openshift version. It might be better to define it as a range with a top end that has been checked by QE. For instance we're aware there are API usages being eliminated in k8s 1.22/ ocp 4.9, if you're sure bundles created with this specified versions value are okay with 4.9 you might consider making the value, `v4.7-v4.9`, or if the bundles will have dependencies on APIs that are going away after v4.8, perhaps use: `v4.7-v4.8`. Setting only a minimum value is a bit of a risk as it may overstate the future compatibility of the bundles produced.

Also, I think this is an SDK project and as such, bundle.Dockerfile is overwritten when `make bundle` is run. You will need to make these changes to `bundle/metadata/annotations.yaml` in this case.

See https://docs.engineering.redhat.com/display/CFC/Delivery for details

Reply:
@bentito, I checked the `annotations.yaml` and it looks right. Was there anything specifically that is incorrect?

Reply:
I don't see any mention of it here: https://github.com/quay/quay-bridge-operator/blob/e6a54c84574bd2ce45a3e183171d7ea511bf57e8/bundle/downstream/metadata/annotations.yaml ? If `make bundle` is in use (likely) then bundle.Dockerfile is overwritten.

Also are you okay with just specifying the minimum version?

Reply:
Discussed this with @ricardomaraschini and we decided to handle the overwriting of manifests in a separate PR

Reply:
> Also are you okay with just specifying the minimum version?
I think it's better if defined as a range `v4.7-v4.9`. Updated it